0000-template.md

```diff
@@ -91,6 +93,23 @@ it does not on its own motivate an RFC.  Please also take into consideration
 that Fabric sometimes intentionally diverges from common distributed
 ledger/blockchain features.
 
+# Testing
```

Comment:
I understand we don't want the RFCs to become a hodgepodge of hand-wavy suggestions, but this passage in particular strikes me as too much process, and makes the RFC process a bit too onerous. It'd probably be more useful to discuss these issues when we go about the RFC, rather than making them a prerequisite for a proposal.

Reply:
^^ This is meant to apply to lines 96-112.

Reply:
This is passage is to make sure that we are not missing dependencies and understand complications of the suggested RFC. Though I understand this might feel too much process this is quite common, for example this is approach adopted for many years in JEP (Java Enhancement Proposal).